src/rules/selector-pseudo-class-whitelist/__tests__/index.js

```diff
@@ -0,0 +1,106 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [[
+    "hover",
+    "nth-child",
+    "root",
+    "placeholder",
+  ]],
+  skipBasicChecks: true,
```

Comment:
Yep, `:global` -https://github.com/stylelint/stylelint/blob/master/src/testUtils/basicChecks.js#L15-L18

It surprised me too. Perhaps we should remove it from the basicChecks?